app/controllers/affiliates_controller.rb

```diff
@@ -0,0 +1,69 @@
+class AffiliatesController < ApplicationController
+  before_action :set_affiliate, only: [:show, :edit, :update, :destroy]
+
+  # GET /affiliates
+  def index
+    @affiliates = Affiliate.all
+  end
+
+  # GET /affiliates/1
+  # TODO: should all coordinators, students, and tutors within this affiliate
+  def show
+    @affiliate = Affiliate.find(params[:id])
```

Comment:
It can. But I was thinking when an user opens "Affiliate" page, he/she should also be able to see more than just the info of this current affiliate. (i.e: coordinators, student,s and tutors belong to this affiliate). I want to do those after I finish writing tests. So i separated "show" from "set_affiliate"